Pima indians diabetes and linear classifier/Заключение/task.md

```diff
@@ -0,0 +1,7 @@
+In this lesson, we've discussed an example of implementing the linear classifier utilizing the stochastic gradient descent.
```

Comment:
Maybe:
In this lesson, we've discussed an example of implementing a linear classifier with stochastic gradient descent.

Reply:
yes, sure